android/src/main/java/com/newrelic/capacitor/plugin/NewRelicCapacitorPluginPlugin.java

```diff
@@ -499,7 +499,9 @@ public void recordError(PluginCall call) {
         stack = (stack == null) ? "null" : stack;
 
         StackTraceElement[] stackFrames = parseStackTrace(stack);
-
+        JSONObject customAttributes = call.getObject("attributes");
+        Map yourHashMap = new Gson().fromJson(String.valueOf(customAttributes), Map.class);
+        exceptionMap.putAll(yourHashMap);
```

Comment:
We need a null check here since attributes is an optional parameter on the typescript definition. We shouldn't add anything if attributes doesn't exist or is null/undefined. If you run the test suite through Android studio or run `npm run test:android` the test suite should pass.

Reply:
Added a null check to make attributes optional